src/errors.ts

```diff
@@ -52,6 +59,6 @@ export class ConflictingCoinFeaturesError extends BitGoStaticsError {
     super(
       `coin feature(s) for coin '${coinName}' cannot be both required and disallowed: ${conflictingFeatures.join(', ')}`
     );
-    Object.setPrototypeOf(this, InvalidContractAddress.prototype);
+    Object.setPrototypeOf(this, InvalidContractAddressError.prototype);
```

Comment:
Oops, this is a pre-existing bug. Please change to `ConflictingCoinFeaturesError.prototype`.

Reply:
fixed